### ISSUE TITLE
How can I get the 1.951 validation EPE with FlowNetS?  (my EPE is 2.03)

### ISSUE THREAD

**Issue by sczhou** [ClementPinard/FlowNetPytorch#30]
Hi, I trained the FlowNetS with default options, but I can not get 1.951 validation EPE,  and my EPE is 2.03. Is that reasonable? And is your split-value set to 0.8?

Thanks!
Shangchen

**Comment by ClementPinard** [comment]
Hello,

Default parameters are here to match original FlowNet training implementation

However, I did some hyperparameter changes to get to 1.951, have you tried the ones i used in [here](https://github.com/ClementPinard/FlowNetPytorch#training-results) ? (among others, epoch size is extended to 2.5k instead of 1k)

I did not use a split value, I used the split file provided [here](https://lmb.informatik.uni-freiburg.de/resources/datasets/FlyingChairs/FlyingChairs_train_val.txt), test set is actually much smaller than 20% of dataset, which can also explain the accuracy drop you got. (but 2.03 is actually pretty good, since it outperforms initial scores reported by FlowNetC in [2015 paper](https://lmb.informatik.uni-freiburg.de/Publications/2015/DFIB15/) )

**Comment by sczhou** [comment]
Yeah, I tried the hyperparameters as you mentioned in your training results. Then, I'll try to use the split file instead. However, there is still a problem in making data list in flyingchairs.py. you use 'glob.iglob(os.path.join(dir,'*_flow.flo'))' to load the flo file list, but I found the iglob's return is disorder, which does not match with the FlyingChairs_train_val.txt.

Thanks for your kind reply, 
Regards,
Shangchen

**Comment by ClementPinard** [comment]
Did not see that ! thanks for poiting it out ! A patch is incoming

**Comment by sczhou** [comment]
Ok, thanks!

**Comment by ClementPinard** [comment]
Done
